.github/workflows/java-ci.yml

```diff
@@ -107,7 +107,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        jvm: [8, 11, 17]
+        jvm: [17]
```

Comment:
This is a bit more of a serious change since we won't be supporting JVM 8 and 11 if we do this